dashboard/migrations/0017_auto_20200720_1203.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.14 on 2020-07-20 16:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0015_auto_20200116_1408'),
+    ]
+
+    operations = [
+        # We have to remove all rows from resource and resource access before performing this migration
+        # As there is no way of knowing which course_id a resource access was linked to
+        # This is the current process for the cron anyway
+        migrations.RunSQL("DELETE FROM resource_access;"),
+        migrations.RunSQL("DELETE FROM resource;"),
```

Comment:
Have we typically worried about doing data manipulation in migrations in the past for MyLA? Seems like usually we just determine whether the database needs to be emptied before running migrations. Maybe this is smart, maybe it's unnecessary. Might be kind of nice to just have this migration be fully auto-generated. I'm okay either way, just thought I'd like to hear your thinking.

Reply:
I'm sure we could have handled it, but we are also not the only school exclusively running this anymore so I figured we need to have everything automated. Having it like this just seemed to make it easier.

If these tables aren't emptied the alternative is to UPDATE the course_id in resource_access from resource.

However resource_access before only had `resource_id + user_id` so there wasn't 100% chance that the match would work. I think it was initially implemented on the wrong table. This is likely a *very* low case (just lecture capture really) so we could have updated and retained the data but I figured since the process as-is currently deletes and re-populates, one more time would be fine.